app/assets/stylesheets/forms.scss

```diff
@@ -67,3 +67,14 @@
 .vertical-drag {
   cursor: ns-resize;
 }
+
+.inset-text-field {
+  border-left: 5px solid #bcbcbc;
+  padding: 6px 0 6px 10px;
+  position: relative;
```

Comment:
You shouldn't need this line now that you've got a margin-left on this.

Reply:
Good spot - thanks - fixed up.